data/brands/amenity/cafe.json

```diff
@@ -589,6 +589,21 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Nero Express (Caffè Nero)",
+      "locationSet": {
+        "include": ["gb", "ie", "tr", "us"]
+      },
+      "tags": {
+        "amenity": "cafe",
+        "brand": "Caffè Nero",
+        "brand:wikidata": "Q675808",
+        "brand:wikipedia": "en:Caffè Nero",
```

Comment:
Does this need a discrete Wikidata/Wikipedia entry from the original/bigger one (if so the CI should fail too)?

Reply:
The branding for Caffè Nero and Nero Express looks similar so it should be fine to use the same Wikidata/Wikipedia entry, it's not required to have a unique Wikidata item for each entry in NSI